### PR TITLE
[#10569] Missing loading icon when adding new question from template

### PR DESCRIPTION
**Pull request by samuelfangjw**
Part of #10569 

**Outline of Solution**

I added a boolean field `isAddingFromTemplate`. A loading spinner is displayed when `isAddingFromTemplate` is true. The spinner is displayed until all questions have been added.

https://user-images.githubusercontent.com/60355570/122561225-2f1ac780-d074-11eb-90f7-9012e9bed0ab.mov

### CODE REVIEW

**Review by madanalogy — APPROVED** [review]
Thanks for the clean implementation!

**Review by rrtheonlyone — CHANGES_REQUESTED** [review]
Any reason why the `isAddingFromTemplate` boolean is parked under `SessionEditFormModel`? 

The edit form modal should not need to be aware of adding from templates unlike the copying/saving/editing/deleting.

**Comment by samuelfangjw** [comment]
> Any reason why the `isAddingFromTemplate` boolean is parked under `SessionEditFormModel`?
> 
> The edit form modal should not need to be aware of adding from templates unlike the copying/saving/editing/deleting.

You're right, no reason for it to be there. I've moved it out of the form model.

File: src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts [hunk 1]
```diff
@@ -81,6 +81,7 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
   courseId: string = '';
   feedbackSessionName: string = '';
   isEditingMode: boolean = false;
+  isAddingFromTemplate: boolean = false;
```
Comment (rrtheonlyone) [inline src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts:84]: Please move this to line 172 or so together with the other booleans

**Review by rrtheonlyone — APPROVED** [review]
Yes, this is good. 

We can merge after a really minor fix below:

File: src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts (hunk 1 above)
Comment (samuelfangjw) [reply src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts]: Thanks for the review! I've moved it over together with the other booleans.

**Pull request merged**